services/graphql-server/src/graphql/utils/embedded-image-tags.js

```diff
@@ -20,16 +26,16 @@ module.exports = async (body, { imageHost, basedb }) => {
       tag.setValid(false);
       return tag;
     }
-    // const defaultSize = ['left', 'right'].includes(tag.get('align')) ? '320' : '640';
-    // const size = tag.get('size', defaultSize).replace('w', '');
-    // @todo Adjust this. Hardcoding for now to allow for crisp images until proper w/h is handled.
-    const size = '1440';
 
     tag.set('alt', createAltFor(image));
     tag.set('src', createSrcFor(imageHost, image, {
-      w: size,
-      fit: 'max',
-      auto: 'format',
+      ...defaults,
+      ...imageAttrs,
+    }));
+    tag.set('srcSet', createSrcFor(imageHost, image, {
```

Comment:
Can you also verify that the site will properly translate the src set from the tag to the HTML element?